includes/datastream.inc

```diff
@@ -138,6 +143,7 @@ function islandora_rest_datastream_post_response(array $parameters) {
   $datastream->label = isset($request['label']) ? $request['label'] : $file['name'];
   $datastream->versionable = isset($request['versionable']) ? (bool) $request['versionable'] : TRUE;
   $datastream->mimetype = isset($request['mimeType']) ? $request['mimeType'] : $mime_detect->getMimetype($file['name']);
+  $datastream->checksumType = isset($request['checksumType']) ? $request['checksumType'] : 'DISABLED';
```

Comment:
Yeah, I would agree with that. This is really just a RESTful wrapper on top of Tuque, so anything Tuque can do it should probably be able to do.

I'd say just pop that default over to `islandora_checksum_checksum_type` and this is good to merge.